### PR TITLE
AddEven

### PR DESCRIPTION
**Pull request by TomaszWeglinski2**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: add-even
🏅 Granted 5 XP for @TomaszWeglinski2

**Comment by ziobron** [comment]
⏱ 3 XP extra for @TomaszWeglinski2 for on-time delivery

File: homework/add-even/addEven.hpp [hunk 1]
```diff
@@ -2,7 +2,12 @@
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+        int sum = 0;
+        for( auto el: numbers ){
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:6]: add space after for loop. But remove after (

for (auto el : number) {

File: homework/add-even/addEven.hpp [hunk 2]
```diff
@@ -2,7 +2,12 @@
...
+            if (el%2 == 0){
+                sum+=el;
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:8]: Add space between operations like '+' '-' '!=' '==' '%' etc...
Add space after while, if, else etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**